pyproject.toml

```diff
@@ -49,6 +49,7 @@ python-magic = { version = ">=0.4.15", optional = true }
 pytz = { version = ">=2019.3", optional = true }
 requests = { version = ">=2.21.0", optional = true }
 requests-oauthlib = { version = ">=1.3.0", optional = true }
+urllib3 = { version = "^2.2.3", optional = true }
```

Comment:
Can we go to >= 2.2.3 or is there a known incompatibility?

Reply:
Also, requests already requires urllib3: https://github.com/psf/requests/blob/main/setup.py#L39

Do we use anything directly from urllib3?

Reply:
@Kircheneer It's used for disabling SSL verification warnings. As for the version used, I just added with `poetry add urllib3` so it was poetry that decided on that format.

Reply:
I know that this is the poetry default, but I don't like it, unless there is a reason to believe we can't use urllib 3.

Reply:
Updated to >=2.2.3.